setup.py

```diff
@@ -975,6 +975,12 @@ def main():
     extras_require = {
         'opt-einsum': ['opt-einsum>=3.3']
     }
+    if platform.system() == 'linux':
+        triton_pin_file = os.path.join(cwd, ".github", "ci_commit_pins", "triton.txt")
+        if os.path.exists(triton_pin_file):
+            with open(triton_pin_file) as f:
+                triton_pin = f.read().strip()
+                extras_require['dynamo'] = ['pytorch-triton=2.0.0+'+triton_pin[:10]]
```

Comment:
We might want to add `jinja2`/`sympy` etc here.  Should test based on a fresh install that we aren't missing any deps.

Reply:
Plz give me a list of dependencies that dynamo needs (is this part of requirements.txt in the dynamo repo)

Reply:
I think:
```
sympy
filelock
networkx
jinja2
pytorch-triton
```

Reply:
With optional deps on:
```
dill
tabulate
```

Reply:
Can I have minversions? Or just list them as is?

Reply:
Currently, we aren't pinned to specific versions.  We could depend on current version or newer -- or just any version.

Reply:
You've already added `scipy` and `networkx` as unconditional dependency in https://github.com/pytorch/pytorch/pull/86461, which probably needs to be moved to a dynamo one, but I guess it could be done later.